waffle_configs/bedrock-prod.env

```diff
@@ -36,9 +36,11 @@ SWITCH_WELCOME_8_EXPERIMENT=on
 SWITCH_WHATSNEW_FIREFOX_98_TURNING_RED=off
 SWITCH_VPN_FUNNEL_OPT_SUBSCRIPTION_URL=on
 SWITCH_BROWSER_MR2_PROMO=on
-SWITCH_HOMEPAGE_MR2_PROMO=on
+SWITCH_HOMEPAGE_MR2_PROMO=off
```

Comment:
This will disable the MR2 promo on the global home page. I think it needs to remain on, and the Fx100 promo will still appear on the English home page because that template doesn't have the other switch.

Reply:
oh! right.